bundles/org.openhab.binding.solarman/src/main/resources/OH-INF/i18n/deyelogger.properties

```diff
@@ -0,0 +1,3 @@
+# FIXME: please add all English translations to this file so the texts can be translated using Crowdin
```

Comment:
I would expect this file to be named solarman.properties and also contain actual default translation 

_to generate the content of this file run: mvn i18n:generate-default-translations_